.github/azure-pipelines.yaml

```diff
@@ -0,0 +1,31 @@
+trigger:
+- master
+
+pr:
+- master
+
+resources:
+  repositories:
+  - repository: azure_template
+    type: github
+    name: mdolab/.github
+    endpoint: mdolab
+
+pool:
```

Comment:
Can this be specified in the template?

Reply:
Yes, it can be moved to be job-specific. I will add this change in the next push.